lnwallet/channel.go

```diff
@@ -551,6 +551,13 @@ func (lc *LightningChannel) diskHtlcToPayDesc(feeRate chainfee.SatPerKWeight,
 		remoteOutputIndex = htlc.OutputIndex
 	}
 
+	customRecords := htlc.CustomRecords.Copy()
+	entryType := Add
+
+	if shouldSetNoop(customRecords, lc.channelState.ChanType) {
```

Comment:
How about:
`entryType := entryTypeForHtlc(htlc.CustomRecords, lc.channelState.ChanType)`? Then we can keep the `htlc.CustomRecords.Copy(),` below and reduce the diff a bit, since we do this at least three times.